### PR TITLE
CLDC-3405-update-retirement-age-validation

### PR DESCRIPTION
**Pull request by arfacamble**
update retirement soft validation pages and copy in en.yml to reflect changes in english retirement age

https://dluhcdigital.atlassian.net/browse/CLDC-3405

### CODE REVIEW

**Comment by RachaelBooth** [comment]
Is there something in the trigger for the soft validation that needs to change as well?

File: spec/models/lettings_log_spec.rb [hunk 1]
```diff
@@ -3382,64 +3382,6 @@ def previous_postcode_fields
...
-      it "returns nil" do
-        expect(lettings_log.retirement_age_for_person_3).to be_nil
-      end
-
-      it "returns the expected plural" do
-        expect(lettings_log.plural_gender_for_person_3).to be_nil
-      end
-    end
-
-    context "when a postcode contains unicode characters" do
```
Comment (RachaelBooth) [inline spec/models/lettings_log_spec.rb:3434]: This looks like an unrelated test that shouldn't have been in this context block - we should probably keep it (but in a better place) unless it's already a duplicate.

File: app/models/log.rb [hunk 2]
```diff
@@ -205,10 +205,6 @@ def blank_invalid_non_setup_fields!
   end
 
   (1..8).each do |person_num|
-    define_method("retirement_age_for_person_#{person_num}") do
-      retirement_age_for_person(person_num)
-    end
-
     define_method("plural_gender_for_person_#{person_num}") do
```
Comment (RachaelBooth) [inline app/models/log.rb:208]: Does this get used anywhere else?

File: spec/models/lettings_log_spec.rb (hunk 1 above)
Comment (arfacamble) [reply spec/models/lettings_log_spec.rb:3434]: good spot!
what a strange place to put that test!

File: app/models/log.rb (hunk 2 above)
Comment (arfacamble) [reply app/models/log.rb:208]: I don't believe so, but it's always impossible to tell for sure with rails since someone _can_ do something very strange like `obj.send(%w(retirement age for person 1).join("_"))`
but I've global searched retirement_age_for which now only gets hits in the old form json files and I can't think of any other way it might be getting called that isn't far fetched.
The tests do all pass as well, so you would hope that that covers things?

Comment (RachaelBooth) [reply app/models/log.rb:208]: Ah, rails... That implies we probably could remove it but it seems right to be wary.

**Review by RachaelBooth — APPROVED** [review]
